tests/unit/Transformer/EmojiStringTransformerCest.php

```diff
@@ -2,13 +2,13 @@
 declare(strict_types = 1);
 
 use Diaclone\Resource\Item;
-use Diaclone\Transformer\StringTransformer;
+use Diaclone\Transformer\EmojiStringTransformer;
 
-class StringTransformerCest
+class EmojiStringTransformerCest
```

Comment:
Does this leave us without a test for `StringTransformer`?

Reply:
yes. i don't think we ever had a string cest